src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -609,6 +610,10 @@ public static void registerItems(final RegistryEvent.Register<Item> event)
 		registerItemBlock(registry, GREEN_STONE_BRICK_SALAMANDER_RIGHT, MSItemGroup.MAIN);
 		registerItemBlock(registry, GREEN_STONE_BRICK_SKAIA, MSItemGroup.MAIN);
 		registerItemBlock(registry, GREEN_STONE_BRICK_TURTLE, MSItemGroup.MAIN);
+		registerItemBlock(registry, SANDSTONE_COLUMN, MSItemGroup.LANDS);
+		registerItemBlock(registry, CHISELED_SANDSTONE_COLUMN, MSItemGroup.LANDS);
+		registerItemBlock(registry, RED_SANDSTONE_COLUMN, MSItemGroup.LANDS);
+		registerItemBlock(registry, CHISELED_RED_SANDSTONE_COLUMN, MSItemGroup.LANDS);
```

Comment:
This is subjective but do you prefer putting these in the LANDS creative tab over the MAIN for a specific reason?

Reply:
Seemed like most new blocks used in lands (as I assume these will be) were in the LANDS tab. There weren't any other new sandstone blocks aside from ores, but the rest of the pink stone blocks were also in LANDS.

Reply:
For that reason it seems appropriate to use LANDS although often blocks/items that can be acquired in the Overworld are put in MAIN

Reply:
Should I change the sandstone columns to be in MAIN, then?

Reply:
I will leave it entirely up to personal preference here

Reply:
Because I suspect that these were added primarily to be used by land structures, I think it makes more sense to keep them in the lands category.